### ISSUE TITLE
Implement Image properties

### ISSUE THREAD

**Issue by grzegorzdec** [magic-script/magic-script-components#227]
| Name               | Type    | Default Value | Description                                                                                                                                                     |
| :----------------- | :------ | :-----------: | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| textCoords         | [vec2]  |               | Sets the texture coordinates used by the image. The texture coordinates must be a list of four coordinates representing the UV coordinates of the four corners. |
| opaque             | boolean |               | Sets the opacity state of the image.                                                                                                                            |
| ui                 | boolean |               | Sets whether the image is rendered on UI Layer.                                                                                                                 |
| imageFrameResource | BigInt  |               | Sets the model resource used for the image frame. Note: Frame models are expected to be one meter high, one meter wide, and centered at the origin.             |
| renderResource     | BigInt  |               | Sets the render resource used for the image.                                                                                                                    |

**Comment by grozdanov** [comment]
MXSCOMP-82

**Comment by grzegorzdec** [comment]
textCoords - textures are not supported by android
opaque - <1 
ui - unclear
imageFrameResource - blocked by another ticket (not implemented yet)
renderResource - as above
